etc/config/sail.amazon.properties

```
@@ -0,0 +1,21 @@
+compilers=&sail
+
+defaultCompiler=sail_0_18
+
+group.sail.groupName=Sail
+group.sail.compilers=sail_0_18
+group.sail.isSemVer=true
+group.sail.baseName=sail
+
+group.sail.name=Sail
+group.sail.exe=sail
+group.sail.versionFlag=--version
+group.sail.options=-c
+group.sail.supportsBinary=true
+group.sail.supportsExecute=true
+group.sail.compilerType=sail
+group.sail.objdumper=objdump
```

Can you specify the full path to objdump please; and I think you can specify it globally (maybe put this on line 4):

```suggestion
objdumper=/opt/compiler-explorer/gcc-14.2.0/bin/objdump
```

---

Will do.

---

Done